src/shared/components/NotFound.tsx

```diff
@@ -134,63 +124,41 @@ const NotFoundNew: FC = () => (
   </AppWrapper>
 )
 
-const NotFoundOld: FC = () => (
-  <div className="container-fluid" data-testid="not-found">
-    <div className="panel">
-      <div className="panel-heading text-center">
-        <h1 className="deluxe">404</h1>
-        <h4>Bummer! We couldn't find the page you were looking for</h4>
-      </div>
-    </div>
-  </div>
-)
+const NotFound: FC = () => {
+  const [isFetchingOrg, setIsFetchingOrg] = useState(false)
+  const location = useLocation()
+  const history = useHistory()
+  const org = useSelector(getOrg)
 
-class NotFound extends Component<Props> {
-  state = {
-    isFetchingOrg: false,
-  }
-
-  async componentDidMount() {
-    if (isFlagEnabled('deepLinking')) {
-      let org = this.props?.org
-
-      if (!org) {
-        this.setState({isFetchingOrg: true})
-        org = await fetchOrg()
-      }
-
-      const deepLinkingMap = buildDeepLinkingMap(org)
-
-      if (deepLinkingMap.hasOwnProperty(this.props.location.pathname)) {
-        event('deeplink', {from: this.props.location.pathname})
-        this.props.history.replace(deepLinkingMap[this.props.location.pathname])
-        return
-      }
-      this.setState({isFetchingOrg: false})
+  const handleDeepLink = useCallback(async () => {
+    if (!org) {
+      setIsFetchingOrg(true)
+      await fetchOrg()
```

Comment:
What's the reason not to use the `org` value returned from `fetchOrg`? The idea was when a logged out user hit this page and then logs in, they would be immediately redirected to the link they were trying to go to. It looks like this will cause unecessary re-rendering since the component will have to wait for redux to be populated with `org`

Reply:
Yeah, you're right it's kind of odd. I think I meant to add that in afterwards but never got around to it. I think it's related to the fact that we have a `GetOrganizations` that dispatches the action after `getOrganizations` is called.